crates/bevy_remote/src/builtin_methods.rs

```diff
@@ -348,7 +364,7 @@ pub fn process_remote_get_request(In(params): In<Option<Value>>, world: &World)
 
 /// Handles a `bevy/get+watch` request coming from a client.
 pub fn process_remote_get_watching_request(
-    In(params): In<Option<Value>>,
+    In((_, params)): In<RemoteWatchingSystemParams>,
```

Comment:
Why are we providing the `watch_id` if they are not used anywhere in the watch systems?
In case they might be used in the future?

Reply:
The idea is to give watch handlers a way to identify _which_ watcher it is handling, giving them a unique key for use with any needed state/cache that needs to be handled on watcher level, rather than method level.

As you’ve noticed it is not used by any of the built in methods, as they are not stateful. But I have encountered the need for watcher state while building some custom methods.

I’m not completely certain this is the best solution though. There are other ways. For example each watcher could have their own SystemState, allowing use of Local<T> instead of managing state using ids and hash maps. Another benefit is cleanup would be automatic.